apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -63,20 +65,20 @@
 @EnableScheduling
 public class SpringConfiguration extends BaseConfiguration {
   /**
-   * The {@link String } Boolean property that is used to enable the fake drug code (00000-0000)
-   * that is used for integration testing. When this property is set to the string 'true', this fake
-   * drug code will be appended to the drug code lookup map to avoid test failures that result from
-   * unexpected changes to the external drug code file in {@link
+   * The {@link String } property that is used to hold drug code file name that is used for
+   * integration testing. When this property is set to the string 'true', this fake drug code will
+   * be appended to the drug code lookup map to avoid test failures that result from unexpected
+   * changes to the external drug code file in {@link
    * FdaDrugCodeDisplayLookup#retrieveFDADrugCodeDisplay}. This property defaults to false and
    * should only be set to true when the server is under test in a local environment.
    */
-  public static final String PROP_INCLUDE_FAKE_DRUG_CODE = "bfdServer.include.fake.drug.code";
+  public static final String PROP_DRUG_CODE_FILE_NAME = "bfdServer.drug.code.file.name";
 
   /**
-   * The {@link String } Boolean property that is used to enable the fake org name that is used for
-   * integration testing.
+   * The {@link String } property that represents org file name that is used for integration
+   * testing.
    */
-  public static final String PROP_INCLUDE_FAKE_ORG_NAME = "bfdServer.include.fake.org.name";
+  public static final String PROP_ORG_NAME = "bfdServer.org.name";
```

Comment:
Might be good to make it clear that this is also a file name
```suggestion
  public static final String PROP_ORG_FILE_NAME = "bfdServer.org.file.name";
```